release: 10.0.0 (#3526)
rome/tools: editors/vscode/README.md

Adds support for the Rome language server to provide formatting, diagnostics, and code actions.

## How the extension works

The extension acts as a language server, which means that you will get all the features
provided by the extension (formatting, diagnostics, etc.) for any file supported by Rome.

The extension will also try to discover the `rome.json` file, which needs to be placed

> at the root of your workspace.

Reviewer: The extension automatically discovers the `rome.json` file in the workspace's root directory.

Reviewer: We should add a section explaining how to make the formatter our default formatter or link to the documentation

## Getting started

...

Author: I was planning to have this section in the website, but I leave that to you. So, what should I do? Add it here?

## Known limitations

> There's a number of known limitations that the team plans to remove in the following releases.

Reviewer: ```suggestion
There are a few limitations that the team plans to remove in the following releases.
```

> - at the moment the discovery of `rome.json` is only limited to the root of the workspace;

Reviewer: ```suggestion
- the discovery of the `rome.json` is limited to the root of the workspace
```

> - changes to the configuration `rome.json` won't be picked automatically;  

Reviewer: ```suggestion
- You need to restart VS Code or Rome's language server for configuration changes in the `rome.json` to take effect.
```

I still hope that some one us has time to fix this by handling the configuration changes in the VS code extension

> - when updating `rome` inside you `package.json`, the rome server needs to be restarted;

Reviewer: ```suggestion
- You must restart VS Code after upgrading `rome` to a new version.
```

- if you don't want Rome's diagnostics in projects that don't use Rome, you need to disable

> the extension

Reviewer: - The Rome extension runs on all projects, even if they don't have a `rome.json` file. That's why you must disable the Rome extensions for workspaces where you don't want Rome enabled.

Adds support for the Rome language server to provide formatting, diagnostics, and code actions.

> ## How the extension works

Author: I applied this changes as part of https://github.com/rome/tools/pull/3284
